src/model/platform-setup/setup-mac.ts

```diff
@@ -72,6 +72,26 @@ class SetupMac {
     return '';
   }
 
+  private static getArchitectureParameters(): string[] {
+    const architectureArgument = [];
+
+    // @ts-ignore
+    const process = require('process');
```

Comment:
I'm not sure but I don't think you need to import process and use TS ignore. Node has process as a global.

Reply:
Oh, alright! I'll try with the change

Reply:
Changed made, and it works too!